bundles/org.smarthomej.binding.telenot/src/main/java/org/smarthomej/binding/telenot/internal/handler/TelenotBridgeHandler.java

```diff
@@ -284,12 +284,18 @@ private void readerThread() {
                             TelenotThingHandler.readyToSendData.set(true);
                             logger.trace("Ready to send data");
                             break;
-                        case INVALID:
+                        case UNKNOWN:
```

Comment:
What is the difference between INVALID and UNKNOWN? I'm asking because one is logged at WARN and the other at DEBUG. What is the user supposed to do when he encounters an UNKNOWN message type?

Reply:
Today I updated my live system to oh3.2
At the start I received an incomplete message (INVALID)
Every valid message begins with "6868" and ends with "16"
This invalid message did not contain it.

However, some messages (even valid ones) may be unknown if the msgType is not currently implemented.

I hope you are okay with this.

Reply:
The question is why you received invalid/incomplete messages. This should be investigated. I'll merge that here anyway, sounds good.